graph_creator/models.py

```diff
@@ -3,23 +3,124 @@
 """
 
 from __future__ import unicode_literals
-from datetime import datetime
+import datetime
+import json
 
 from django.db import models
+from django.db.models import Sum, Count, DateField
+from django.db.models.functions import Trunc
+
+
+def get_previous_day_start_and_end_dates():
+    current_datetime = datetime.datetime.today() - datetime.timedelta(days=1)
+    start_of_day = current_datetime.replace(hour=0, minute=0, second=0, microsecond=0)
+    end_of_day = current_datetime.replace(hour=23, minute=59, second=59, microsecond=59)
+
+    return start_of_day, end_of_day
 
 
 class DataStorage(models.Model):
```

Comment:
@z4y4ts, natural key is a secret_token and id (pk) for now. 
I understood your comments, thanks.